.eslintrc

```diff
@@ -20,6 +21,8 @@
     "dot-notation": 0,
     "no-debugger": 2,
     "no-undef": 2,
-    "no-unused-vars": 2
+    "no-unused-vars": 2,
+    "semi": 2,
+    "arrow-parens": [2, "as-needed"]
```

Comment:
Does this config result in auto adding semi-colons, commas, etc on save/prettify?